index.html

```diff
@@ -1,203 +1,356 @@
+                            <li class="mcan" ><a href=""><img class="twitter" src="angellist.png" alt="twitter"></a></li>
+                            <li class="mcan"><a href=""><img class="twitter" src="twitter.png" alt="twitter"></a></li>
+                            <li class="mcan" ><a href=""><img class="twitter" src="medium.png" alt="twitter"></a></li>
+                        </ul>
+                    </div>
+                    <div class="mabout">
+                        <button class="mget">Get my resume</button>
+                    </div>
+                    <div class="mabout">
+                        <ul class="mdown">
+                            <li class="mdown"><div class="mic"><a class="mlang">Languages</a><img class="mvec" src="vect.png"></div>
+                            <li class="mdown"><div class="mic"><div class="mjsfile"><a class="mjs">JavaScript</a></div></div></li>
+                            <li class="mdown"><div class="mic"><div class="mjsfile"></a><a class="mhtml5">HTML</a></div></div></li>
+                            <li class="mdown" ><div class="mic"><div class="mjsfile"></a><a class="mcss3">CSS</a></div></div></li></li>
+                            <li class="mdown"><div class="mic"><a class="mlang">Frameworks</a><img class="mvec-1" src="vect1.png"></div></li>
+                            <li class="mdown"><div class="mic"><img class="mline" src="line.png"></div></li>
+                            <li class="mdown"><div class="mic"><a class="mlang">Skills</a><img class="mvec-1" src="vect1.png"></div></li>
+                            <li class="mdown"><div class="mic"><img class="mline" src="line.png"></div></li>
+                        </ul>
+                    </div>
+                    <article class="mcontact">
+                        <div class="mtopback"><h1 class="mcontact-me">Contact me</h1></div>
+                        <p class="mcontact-page">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                            I’d love to help with it!</p>
+                        <div class="mbottomback">
+                        <form action="https://formspree.io/f/xeqwyeqe" method="post">
+                            <input id="mname" type="text" maxlength="30" name="name" minlength="3" placeholder="name"  required pattern="[A-Z]*[a-z]*">
+                            <input id="memail" type="email"  name="email" placeholder="Email" required>
+                            <textarea id="mmessage" maxlength="500" name="message" placeholder="Write your message here..." required></textarea>
+                            <button class="mtouch" type="submit">Get in touch</button>
+                            </form></div>
+                            <div class="mfoot"></div>
+                    </article>
+                </article>     
+                </section>
+            </section>

```

Comment:
To uphold the DRY principle and make optimal use of media queries, it's advisable to construct the app's HTML file just once. By employing media queries, we can then adapt it for desktop sizes.

Currently, the HTML file redundantly contains the entire code. This redundancy should be avoided. Essentially, the HTML code should remain unchanged from the previous milestones, and we should only introduce the desktop view through media queries in the CSS file.

Once this aspect is fully grasped, we can proceed with future milestones.

- [ ] Please eliminate the section of code in the HTML file that duplicates the entire HTML structure of the project, and rely solely on media queries in the CSS file to differentiate between the mobile and desktop views.